uni/lib/view/login/login.dart

```diff
@@ -109,54 +110,51 @@ class LoginPageViewState extends State<LoginPageView> {
-                children: [
-                  Padding(
-                    padding: EdgeInsets.only(
-                      bottom: queryData.size.height / 20,
-                    ),
+          body: Padding(
+            padding: EdgeInsets.only(
+              left: queryData.size.width / 8,
+              right: queryData.size.width / 8,
+            ),
+            child: ListView(
+              children: [
+                Padding(
+                  padding: EdgeInsets.only(
+                    bottom: queryData.size.height / 20,
                   ),
-                  createTitle(queryData, context),
-                  Padding(
-                    padding: EdgeInsets.only(
-                      bottom: queryData.size.height / 35,
-                    ),
+                ),
+                createTitle(queryData, context),
+                Padding(
+                  padding: EdgeInsets.only(
+                    bottom: queryData.size.height / 35,
                   ),
-                  getLoginForm(queryData, context),
-                  Padding(
-                    padding: EdgeInsets.only(
-                      bottom: queryData.size.height / 35,
-                    ),
+                ),
+                getLoginForm(queryData, context),
+                Padding(
+                  padding: EdgeInsets.only(
+                    bottom: queryData.size.height / 35,
                   ),
-                  createForgetPasswordLink(context),
-                  Padding(
-                    padding: EdgeInsets.only(
-                      bottom: queryData.size.height / 15,
-                    ),
+                ),
+                createForgetPasswordLink(context),
+                Padding(
+                  padding: EdgeInsets.only(
+                    bottom: queryData.size.height / 15,
                   ),
-                  createLogInButton(queryData, context, _login),
-                  Padding(
-                    padding: EdgeInsets.only(
-                      bottom: queryData.size.height / 35,
-                    ),
+                ),
+                createLogInButton(queryData, context, _login),
+                Padding(
+                  padding: EdgeInsets.only(
+                    bottom: queryData.size.height / 35,
                   ),
-                  createStatusWidget(context),
-                  Padding(
-                    padding: EdgeInsets.only(
-                      bottom: queryData.size.height / 35,
-                    ),
+                ),
+                createStatusWidget(context),
+                Padding(
+                  padding: EdgeInsets.only(
+                    bottom: queryData.size.height / 35,
                   ),
-                  createSafeLoginButton(context),
-                ],
-              ),
+                ),
+                createSafeLoginButton(context),
+              ],
             ),
-            onWillPop: () => onWillPop(context),
```

Comment:
You can now remove the unused `onWillPop` method.

Reply:
I will do it.

Reply:
And exitApp variables and related methods, too.